platform-monitoring/operator/init.sh

```diff
@@ -41,6 +41,10 @@ sed -i 's/Test-Cluster-TiFlash-Summary/'$TIDB_CLUSTER_NAME'-TiFlash-Summary/g'
 cp /tmp/tiflash_proxy_summary.json $GF_PROVISIONING_PATH/dashboards
 sed -i 's/Test-Cluster-TiFlash-Proxy-Summary/'$TIDB_CLUSTER_NAME'-TiFlash-Proxy-Summary/g' $GF_PROVISIONING_PATH/dashboards/tiflash_proxy_summary.json
 
+# DM
+cp /tmp/dm.json $GF_PROVISIONING_PATH/dashboards
+sed -i 's/Test-Cluster-DM/'$TIDB_CLUSTER_NAME'-DM/g'  $GF_PROVISIONING_PATH/dashboards/dm.json
```

Comment:
The dm cluster name should be used because they may be different.

Reply:
fixed